<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>93</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_DBPath">
       <property name="text">
        <string>Open DataBase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_GetData">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Get Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_auditories">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Auditories</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_workers">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Workers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_auditorysettings">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Auditory Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Database path:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_DBPath">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>351</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>69</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Log:</string>
    </property>
    <widget class="QTextEdit" name="textEdit_log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>171</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
